src/components/AddEditRemoveNav.tsx

```diff
@@ -1,43 +1,78 @@
                 <div className="container-fluid">
-                    <div>
-                        <ul className="nav navbar-nav navbar-right side-nav">
-                            {/*<li className="action-add-button">
-                                <a href='#' onClick={this.props.addAction} title={this.props.addText ? this.props.addText : ""}>
-                                    <i className={`glyphicon glyphicon-plus-sign ${this.props.addAction ? "" : "disabled"}`}></i>
-                                    <i className={`glyphicon glyphicon-plus-sign disabled`}></i>
-                                </a>
-                            </li>*/}
-                            <li className="action-edit-button">
-                                <a href="#" onClick={this.props.editAction} title={this.props.editText ? this.props.editText : ""}>
-                                    {/*<i className={`glyphicon glyphicon-edit ${this.props.editAction ? "" : "disabled"}`}></i>*/}
-                                    <i className={`glyphicon glyphicon-edit disabled`}></i>
-                                </a>
-                            </li>
-                            <li className="action-remove-button">
-                                <a href="#" onClick={this.props.removeAction} title={this.props.removeText ? this.props.removeText : ""}>
-                                    {/*<i className={`glyphicon glyphicon-remove-sign ${this.props.removeAction ? "" : "disabled"}`}></i>*/}
-                                    <i className={`glyphicon glyphicon-remove-sign disabled`}></i>
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
-                    <p className='side-nav-header'>{this.props.title}</p>
+                    <p>{this.props.title}</p>
+                    <NavItem>
+                        <NavLink href="#" onClick={this.props.addAction} title={this.props.addText ? this.props.addText : ""}>
+                            <i className={`glyphicon glyphicon-plus-sign ${this.props.addAction ? "" : "disabled"}`}></i>
+                        </NavLink>
+                    </NavItem>
+                    <NavItem>
+                        <NavLink href="#" onClick={this.props.editAction} title={this.props.editText ? this.props.editText : ""}>
+                            <i className={`glyphicon glyphicon-edit disabled`}></i>
+                        </NavLink>
+                    </NavItem>
+                    <NavItem>
+                        <NavLink href="#" onClick={this.props.removeAction} title={this.props.removeText ? this.props.removeText : ""}>
+                            <i className={`glyphicon glyphicon-remove-sign disabled`}></i>
+                        </NavLink>
+                    </NavItem>
                 </div>
-            </nav>
+            </Nav>
         )
     }
+    // render() {
```

Comment:
Do we want to keep these commented out for now?

Reply:
No, I should have removed that. Its the old non-typescript bootstrap nav elements that were conflicting with the variable edit modal css.